packages/pangea-sdk/pangea/services/authn/authn.py

```diff
@@ -80,15 +98,55 @@ def list(
             order_by: Optional[m.SessionListOrderBy] = None,
             size: Optional[int] = None,
         ) -> PangeaResponse[m.SessionListResults]:
+            """
+            List session (service token)
+
+            List sessions.
+
+            OperationId: authn_post_v1_session_list
+
+            Args:
+                filter (dict, optional):
+                last (str, optional): Reflected value from a previous response to obtain the next page of results.
+                order (m.ItemOrder, optional): Order results asc(ending) or desc(ending).
+                order_by (m.SessionListOrderBy, optional): Which field to order results by.
+                size (int, optional): Maximum results to include in the response. Minimum: 1.
+
+            Returns:
+                A PangeaResponse with a list of sessions in the response.result field.
+                    Available response fields can be found in our
+                    [API Documentation](https://pangea.cloud/docs/api/authn#list-session-service-token).
+
+            Examples:
+                response = authn.session.list()
+            """
             input = m.SessionListRequest(filter=filter, last=last, order=order, order_by=order_by, size=size)
             response = self.request.post("v1/session/list", data=input.dict(exclude_none=True))
             if response.raw_result is not None:
                 response.result = m.SessionListResults(**response.raw_result)
             return response
 
-        # https://dev.pangea.cloud/docs/api/authn#invalidate-all-sessions-belonging-to-a-user
+        # https://pangea.cloud/docs/api/authn#log-out-service-token
         # - path: authn::/v1/session/logout
         def logout(self, user_id: str) -> PangeaResponse[m.SessionLogoutResult]:
+            """
+            Log out (service token)
+
+            Invalidate all sessions belonging to a user.
+
+            OperationId: authn_post_v1_session_logout
+
+            Args:
+                user_id (str): The identity of a user or a service.
```

Comment:
Not sure if identity and id are the same. Are they? I think sometime we had both and they were different stuff